adafruit_magtag/network.py

```diff
@@ -28,9 +28,15 @@
 """
 
 import neopixel
+import microcontroller
```

Comment:
the import for microcontroller can go inside of the try/except block after the typing import. 

This will save some RAM because it won't get imported on microcontrollers where it isn't used at runtime.

Reply:
That makes good sense! Thanks!